.circleci/base-generated-config.yaml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+
+jobs:
+  test-example:
+    parameters:
+      example-app-name:
+        type: string
+      path:
+        type: string
+    machine:
+      image: ubuntu-2004:current
+      docker_layer_caching: true
+    environment:
+      BUILDING_BLOCK: digital-registries
+      TESTED_APPLICATION: << parameters.example-app-name >> 
+      TEST_RESULT_REPORT_FILENAME: example_result.message
+    steps:
+      - checkout
+      - run:
+          name: Execute tests
+          command: |
+            # Execute application test_entrypoint.sh in background. 
+            chmod u+x <<parameters.path >> test_entrypoint.sh &
+            cd <<parameters.path >> 
+            ./test_entrypoint.sh &
+            
+            # Execute gherkin tests for given application 
+            cd ./../../test/
```

Comment:
I'm not a fan of putting paths like `../.././../..`  in config files like this. Can we perhaps use a variable such as $CIRCLE_WORKING_DIRECTORY or something to specify the path here?